src/wallet/rpcwallet.cpp

```diff
@@ -75,6 +75,16 @@ bool EnsureWalletIsAvailable(bool avoidException)
     return true;
 }
 
+bool EnsureHDSeedIsAvailable(bool avoidException = false) {
+    if (!pwalletMain || !pwalletMain->IsHDSeedAvailable()) {
```

Comment:
No need to check `pwalletMain` is null due to we are in wallet code. If it null that mean something wrong.